document/core/util/macros.def

```diff
@@ -1068,17 +1074,19 @@
 
 .. Relaxed Memory Model Judgements
 
-.. |valid| mathdef:: \mathrel{\mbox{valid}}
-.. |validwith| mathdef:: \mathrel{\mbox{valid-with}}
+.. todo:: include xrefs
```

Comment:
Is this related to the CI builds failing?

Reply:
Probably. I moved the todo elsewhere.